CS3247-Game-Development-Team-6/Doodles
# [DRAFT] GUI Update (without spells)

- [ ] Add all spells into preview
- [x] Create new SpiderLevel-beta and TutorialLevel-beta scenes
- [ ] Link the indicators to the managers

To test:

Make sure Build settings contains the following scenes:
![image](https://user-images.githubusercontent.com/23572486/195966971-a994c44c-c40d-4927-855b-83d6e50bcb0f.png)

1. Load the Menu-beta scene
2. Click on "LEVELS" and try loading a level ("CONTINUE" and "NEW GAME" currently still go to the old loadout)
3. Try visualizing the
